src/Morphir/Scala/Feature/Core.elm

```diff
@@ -337,6 +337,20 @@ mapCustomTypeDefinition currentPackagePath currentModulePath moduleDef typeName
                         )
                     ]
 
+                else if List.length ctorArgs == 0 then
+                    [ Scala.withoutAnnotation
+                        (Scala.TypeAlias
+                            { alias = typeName |> Name.toTitleCase
+                            , typeArgs = []
+                            , tpe = Scala.TypeRef [ typeName |> Name.toTitleCase ] "type"
+                            }
+                        )
```

Comment:
Can you put a comment into the code explaining why you did this? That hard-coded "type" value looks suspicious.